src/middleware/protobuf/gpsd.proto

```diff
@@ -0,0 +1,148 @@
+syntax = "proto2";
+
+import "goby/middleware/protobuf/geographic.proto";
+import "dccl/option_extensions.proto";
+
+package goby.middleware.protobuf.gpsd;
+
+
+
+message Satellite {
+  option (dccl.msg).unit_system = "si";
+  required int32 prn = 1;
+  optional double az = 2
+    [(dccl.field) = {units {
+derived_dimensions: "plane_angle"
+                      system: "angle::degree"
+    }}];
+  optional double el = 3
+    [(dccl.field) = {units {
+derived_dimensions: "plane_angle"
+                      system: "angle::degree"
+    }}];
+  optional double ss = 4;
+  optional bool used = 5;
+  optional int32 gnssid = 6;
+  optional int32 svid = 7;
+  optional int32 sigid= 8;
+  optional int32 freqid = 9;
+  optional int32 health = 10;
+}
+
+message SKY {
```

Comment:
What about SKYView, ATTitude and TimePositionVelocity instead of SKY,ATT,TPV for protobuf message names? I'd like to make the message names more clear to a casual (non-GPSD) reader, but I understand the value of the abbreviations when referencing GPSD docs. Happy to discuss this one and hear your thoughts.

Reply:
That seems fine. We can even do away with the capital first three. It doesn't really add much.

Reply:
Want me to change the names of the goby groups as well?